src/realm/spec.hpp

```diff
@@ -100,7 +100,7 @@ class Spec {
     ArrayStringShort m_names; // 2nd slot in m_top
     Array m_attr;             // 3rd slot in m_top
                               // 4th slot in m_top not cached
-    Array m_enumkeys;         // 5th slot in m_top
+                              // 5th slot in m_top
```

Comment:
I'd suggest adding to the comment something like this: "5th slot in m_top, old enum keys which was never released"